tests/test_pdf_polynomials.py

```diff
@@ -0,0 +1,28 @@
+#  Copyright (c) 2019 zfit
+
+
+import pytest
+import zfit
+from zfit import ztf
+
+from zfit.core.testing import setup_function, teardown_function, tester
+
+obs1 = zfit.Space(obs="obs1", limits=(-0.9, 0.85))
+
+
+def test_legendre_polynomial():
+    legendre = zfit.pdf.Legendre(obs=obs1, degree=5)
+
+    integral = legendre.integrate(limits=obs1)
+    assert pytest.approx(1, rel=1e-3) == zfit.run(integral)
```

Comment:
Why is the integral 1 with the observable in a limited range? Am I misunderstanding something?

Reply:
By definition: it's the integral over the same range as the normalization range. But we don't need it here, it was more of an "make sure it runs without obvious errors" test, that the integral calculations don't break.